packages/oracle-node/manifests/data-services/primary.json

```diff
@@ -17,6 +17,7 @@
     "USDC": {
       "source": [
         "binance",
+        "binance-ws",
```

Comment:
I added just a couple of symbols as a proof of concept, even though many more are possible. Hope that's fine.